third_party/airflow/README.md

```diff
@@ -36,6 +36,16 @@ python3.8 -m pip install armada-airflow
 
 From the top level of the repo, you should run `make airflow-operator`.  This will generate proto/grpc files in the jobservice folder.
 
+Airflow with the Armada operator can be run alongside the other Armada services via the localdev docker-compose 
```

Comment:
When you refer to an "armada operator" here, what do you mean?

Reply:
well Armada operator is the airflow plugin that we provide. So this runs an airflow instance (other software) with our operator installed.

Reply:
Do we maybe want to be more specific to what we mean hear since we are building an armada operator seperately to handle installs?

At the very least create an issue to track that this change will be needed to our docs in future.